cmd/influx/bucket.go

```diff
@@ -181,15 +181,15 @@ func bucketFindF(cmd *cobra.Command, args []string) error {
 		"ID",
 		"Name",
 		"Retention",
-		"Organization",
+		"Org",
 		"OrganizationID",
 	)
 	for _, b := range buckets {
 		w.Write(map[string]interface{}{
 			"ID":             b.ID.String(),
 			"Name":           b.Name,
 			"Retention":      b.RetentionPeriod,
-			"Organization":   b.Organization,
+			"Org":            b.Org,
 			"OrganizationID": b.OrganizationID.String(),
```

Comment:
Should this be `OrgID`?

Reply:
Yeah, I believe there is another ticket for that one.